kubernetes/cmsweb/scripts/deploy-secrets.sh

```diff
@@ -1,24 +1,31 @@
 #!/bin/bash
 # helper script to deploy given service with given tag to k8s infrastructure
 
-if [ $# -ne 3 ]; then
-    echo "Usage: deploy-secrets.sh <namespace> <service-name> <path_to_configuration>"
+if [ $# -ne 4 ]; then
+    echo "Usage: deploy-secrets.sh <namespace> <service-name> <path_to_configuration> <reference_of_decryption_barbican_secret>"
     exit 1
 fi
 
 ns=$1
 srv=$2
 conf=$3
+secretref=$4
 
     # cmsweb configuration area
     echo "+++ configuration: $conf"
     echo "+++ cms service : $srv"
     echo "+++ namespaces   : $ns"
+    echo "+++ secretref   : $secretref"
 
     if [ ! -d $conf/$srv ]; then
 	echo "Unable to locate $conf/$srv, please provide proper directory structure like <configuration>/<service>/<files>"
   	exit 1
     fi
+   
+    # Get SOPS decryption key and put it in config directory
+    mkdir -p ~/.config/sops/age/
+    rm ~/.config/sops/age/keys.txt
```

Comment:
I'll remove this line completely, as script should not intervene to do anything in my personal areas. Instead,  put 
```
if [ ! -f $HOME/.openstack/config/sops/age/openstack-keys.txt ]; then
fi
```
block to check for specific file and if necessary download necessary file. Moreover, if we'll need to keep different `$secretref` it would be better to use this name either as part of file or directory name